drivers/encx24j600/include/encx24j600_defines.h

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright (C) 2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     driver_encx24j600
+ * @{
+ *
+ * @file
+ * @brief       Register definitions for the ENCX24J600 Ethernet device
+ *
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef ENCX24J600_REGS_H
+#define ENCX24J600_REGS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name SPI instruction set
+ * @{
+ */
+#define RCR         0x00  /* read control register */
+#define WCR         0x04  /* write control register */
+
+#define RCRU        0x20  /* read control register unbanked */
+#define WCRU        0x22  /* write control register unbanked */
+
+#define BFSU        0x24  /* set bits unbanked */
+#define BFCU        0x26  /* clear bits unbanked */
+
+#define RGPDATA     0x28  /* Read EGPDATA */
+#define WGPDATA     0x2a  /* Write EGPDATA */
+
+#define RRXDATA     0x2c  /* Read ERXDATA */
+#define WRXDATA     0x2e  /* Write ERXDATA */
+
+#define RUDADATA    0x30  /* Read EUDADATA */
+#define WUDADATA    0x32  /* Write EUDADATA */
+
+#define BFS         0x80  /* Bit Field Set */
+#define BFC         0xa0  /* Bit Field Clear */
+
+#define SETETHRST   0xca  /* System Reset */
+#define SETPKTDEC   0xcc  /* Decrements PKTCNT by setting PKTDEC (ECON1<5>) */
+#define ENABLERX    0xe8  /* Enables packet reception by setting RXEN (ECON1<0>) */
+#define DISABLERX   0xea  /* Disable packet reception by clearing RXEN (ECON1<0>) */
+#define SETEIE      0xec  /* Enable Ethernet Interrupts by setting INT (ESTAT<16>) */
+#define CLREIE      0xee  /* Disable Ethernet Interrupts by clearing INT (ESTAT<16>) */
+
+#define B0SEL       0xc0  /* select bank 0 */
+#define B1SEL       0xc2  /* select bank 0 */
+#define B2SEL       0xc4  /* select bank 0 */
+#define B3SEL       0xc6  /* select bank 0 */
+#define RBSEL       0xc8  /* Read Bank Select */
+
+#define SETTXRTS    0xd4  /* Sets TXRTS (ECON1<1>), sends an Ethernet packet */
+
```

Comment:
I'm sorry, I don't share that style preference. If I use newlines to group lines within the doxygen scope, I like that newline at the end of the scope.